backend/test/experiment/experiment_retrieve/RetrieveNonExistentExperiment.test.js

```diff
@@ -0,0 +1,12 @@
+const expect = require('chai').expect;
+const fetch = require('node-fetch');
+
+const SERVER = 'http://localhost:8000/api';
+
+describe('RetrieveNonExistentExperiment', function () {
+    it('GET /experiments/:id with an invalid id must not retrieve an experiment.', async () => {
+
+        const response = await fetch(SERVER + '/experiments' + '/0');
+        expect(response.status).to.be.equal(400);
```

Comment:
Please check for the error message here too.

Reply:
Done! Thanks.